.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
Is it possible to associate this with a specific line of code or does it have to exempt the whole file?

Reply:
Good catch.